content/md/products-super-power/manage-your-views.md

```diff
@@ -22,7 +22,7 @@ To create your first view:
 ![Create a view](../img/Products_CreateView.png)
 1.  Select `Create view` in the dropdown menu. A window will appear.
 1.  Enter a name for your view
-1.  By default, the option *`I want to create a personal view`* is selected. If you want to share this view with every user, just uncheck this box.
+1.  By default, the option *`I want to create a private view`* is selected. It means that the view will be only visible by you. If you want to share this view with every user, just uncheck this box.
```

Comment:
`It means that the view will be only visible by you` seems strange to me.  
Maybe `It means that the view will be visible only for you`?  
To check with an english master

Reply:
`will only be visible by you`  or `will be visible only by you`

Reply:
merci! :)